<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Справочник приходных / расходных накладных (№1)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableView" name="tableInvoiceReferences"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Справочник работников (№2)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Справочник товаров (№4)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="change_data_in_table">
      <property name="text">
       <string>Изменить данные в таблице</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableView" name="tableEmployeeReferences"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="read_data_in_file">
      <property name="text">
       <string>Считать данные из файла</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="write_data_in_file">
      <property name="text">
       <string>Записать данные в файл</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Учёт поступления и выдачи товара (№3)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QTableView" name="tableAFT"/>
    </item>
    <item row="8" column="1">
     <widget class="QTableView" name="tableProductReferences"/>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="add_all_tables">
      <property name="text">
       <string>Заполнить все таблицы</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="clear_all_tables">
      <property name="text">
       <string>Очистить таблицы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
